projects/lcms/build.sh

```diff
@@ -93,12 +93,12 @@ make -j$(nproc) all
 # build your Sydr targets
 mkdir /lcms_cov
 OUT=/lcms_cov
-FUZZERS="cms_link_sydr cmsIT8_load_sydr cms_transform_sydr cms_overwrite_transform_sydr"
+FUZZERS="cms_link cmsIT8_load cms_transform cms_overwrite_transform"
 $CC $CFLAGS /opt/StandaloneFuzzTargetMain.c -c -o main.o
 for F in $FUZZERS; do
     $CC $CFLAGS -c -I /lcms/include \
         /lcms/$F.c -o /lcms/$F.o
     $CXX $CXXFLAGS \
-        main.o /lcms/$F.o -o $OUT/$F \
+        main.o /lcms/$F.o -o $OUT/$F\_sydr \
```

Comment:
Rename to _cov (also fix names in toml configs)

Reply:
done